src/main/java/net/sf/jabref/model/cleanup/FieldFormatterCleanups.java

```diff
@@ -0,0 +1,122 @@
+package net.sf.jabref.model.cleanup;
```

Comment:
I have a suggestion regarding this new package: How about moving it into `model.metadata`? After all, the classes in the package are about how the cleanup information is stored in the meta data. What do you think?

Reply:
@calixtus and @koppor currently wonder why this class was moved to `model`. All implementing classes are in the `logic` package: 

![grafik](https://user-images.githubusercontent.com/1366654/77253729-755c5d80-6c5c-11ea-97e9-0ed945d91a73.png)